Attempt to deal with node-typescript being at 6.x in sid but upstream using 7.x

--- a/third_party/devtools-frontend/src/scripts/build/typescript/ts_library.py
+++ b/third_party/devtools-frontend/src/scripts/build/typescript/ts_library.py
@@ -301,12 +301,12 @@ def main():
         tsconfig['compilerOptions']['noEmit'] = True
     tsconfig['compilerOptions']['outDir'] = '.'
     tsconfig['compilerOptions']['tsBuildInfoFile'] = tsbuildinfo_name
-    tsconfig['compilerOptions']['target'] = opts.es_target
+    tsconfig['compilerOptions']['target'] = 'es2022'  #opts.es_target.lower()
     es_libs = ['dom', 'dom.iterable'] if opts.es_libs is None else opts.es_libs
-    tsconfig['compilerOptions']['lib'] = es_libs + [
-        'ES2023', 'ES2024.Promise', 'ESNext.Iterator', 'ESNext.Collection',
+    tsconfig['compilerOptions']['lib'] = list(map(str.lower, es_libs + [
+        'ES2023', 'ESNext.Promise', 'ESNext.Collection',
         'ESNext.Array'
-    ]
+    ]))
 
     if maybe_update_tsconfig_file(tsconfig_output_location, tsconfig) == 1:
         return 1
